src/pages/Notification/NotificationPage.jsx

```diff
@@ -70,32 +185,71 @@ const NotificationPage = () => {
           onTagChange={setTag}
         />
         <SBtnWrapper>

```

Comment:
`ol` 태그를 사용하는 게 더 적절하겠어요~
```suggestion
        <SOl>
```

Reply:
수정했습니다!